activerecord/lib/active_record/migration.rb

```diff
@@ -733,6 +733,24 @@ def reversible
       execute_block { yield helper }
     end
 
+    # Used to specify an operation that is only run when migrating up
+    # (for example, populating a new column with its initial values).
+    #
+    # In the following example, the new column `published` will be given
+    # the value `true` for all existing records.
+    #
+    #    class AddPublishedToPosts < ActiveRecord::Migration[5.3]
+    #      def change
+    #        add_column :posts, :published, :boolean, default: false
+    #        populate do
+    #          Post.update_all(published: true)
```

Comment:
I think using model directly in migration is fragile since future modification on validation or callback may break this.
If a succeeding migration do `drop_table :posts` and `app/models/post.rb` is deleted, `Post` even does not exist.

How about defining model in place?
```
class AddPublishedToPosts < ActiveRecord::Migration[5.3]
  class Post < ActiveRecord::Base; end
  def change
    ....
```

Reply:
That's a good point. Maybe the example and test should use `execute` instead? That could be easily done with:

```ruby
execute "update posts set published = 'true'"
```

Reply:
👍